### PR TITLE
Add documentation for claiming a Docker container at creation

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

Both `docker run` and Docker Compose commands for claiming at creation. Claiming a running container is handled over in the claim document.

##### Component Name

area/packaging

### CODE REVIEW

File: packaging/docker/README.md [hunk 1]
```diff
@@ -94,14 +94,85 @@ volumes:
...
+  -v /etc/group:/host/etc/group:ro \
+  -v /proc:/host/proc:ro \
+  -v /sys:/host/sys:ro \
+  -v /etc/os-release:/host/etc/os-release:ro \
+  --restart unless-stopped \
+  --cap-add SYS_PTRACE \
+  --security-opt apparmor=unconfined \
+  -e NETDATA_CLAIM_TOKEN = "TOKEN" \
+  -e NETDATA_CLAIM_URL = "ROOMS" \
+  -e NETDATA_CLAIM_ROOMS = "https://app.netdata.cloud" \
```
[the remark this answers is not in the document]
Comment (ilyam8) [reply packaging/docker/README.md:126]: Perhaps we just need to make a link to the https://learn.netdata.cloud/docs/agent/claim#using-environment-variables ?

[the remark this answers is not in the document]
Comment (joelhans) [reply packaging/docker/README.md]: All this disparate claiming information is indeed getting extraordinarily complicated. I need to have a think about how else we can present this information for all the different scenarios. It's probably going to need an entire re-think.

[the remark this answers is not in the document]
Comment (ilyam8) [reply packaging/docker/README.md]: >  Claiming a running container is handled over in the claim document.

My point is we have both 
 - _Claim **a running** Netdata Agent container_ : https://learn.netdata.cloud/docs/agent/claim#using-docker-exec
 - _Claim an Agent running in Docker_ : https://learn.netdata.cloud/docs/agent/claim#using-environment-variables

in the https://learn.netdata.cloud/docs/agent/claim

Why we need the same info in this doc and not a link to the claim?

[the remark this answers is not in the document]
Comment (joelhans) [reply packaging/docker/README.md]: I finally see what you mean. I had forgotten that a lot of this information already exists over in the claiming doc. 

I moved a bulk of the info over to `/claim/README.md` with just a link from the Docker packaging doc. Hopefully this helps streamline things a bit.

**Review by ilyam8 — APPROVED** [review]

File: claim/README.md [hunk 2]
```diff
@@ -6,13 +6,12 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/claim/README.md
 
 # Agent claiming
 
-Agent claiming allows a Netdata Agent, running on a distributed node, to securely connect to Netdata Cloud. A Space's
-administrator creates a **claiming token**, which is used to add an Agent to their Space via the [Agent-Cloud link
-(ACLK)](/aclk/README.md).
+Agent claiming allows a Netdata Agent, running on a distributed node, to securely connect to [Netdata
```
Comment (manos-saratsis) [inline claim/README.md:9]: @joelhans Shall we introduce the term connect instead of the term claim?

**Comment by ilyam8** [comment]
Added in #11442 by @hugovalente-pm

**Pull request closed without merging**